### PR TITLE
Various fixes for test app

### PR DESCRIPTION
**Pull request by mickmister**

### CODE REVIEW

File: http.go [hunk 1]
```diff
@@ -96,8 +96,17 @@ type lookupResponse struct {
 }
 
 func handleLookup(items []apps.SelectOption) callHandler {
-	return func(_ *apps.CallRequest) apps.CallResponse {
-		return apps.NewDataResponse(lookupResponse{items})
+	return func(creq *apps.CallRequest) apps.CallResponse {
+		query := strings.ToLower(creq.Query)
```
Comment (mickmister) [inline http.go:100]: This makes it so the lookup calls have the text filtered by matching the user's input text. You'll need to deploy this PR of the apps plugin to use this feature https://github.com/mattermost/mattermost-plugin-apps/pull/304

File: go.mod [hunk 2]
```diff
@@ -11,3 +11,5 @@ require (
 	github.com/mattermost/mattermost-server/v6 v6.0.0-20220315170027-7710a2fe3741
 	github.com/pkg/errors v0.9.1
 )
+
+replace github.com/mattermost/mattermost-plugin-apps v0.7.1-0.20220114173732-f10616dc8752 => ../mattermost-plugin-apps
```
Comment (levb) [inline go.mod:15]: remove?

**Review by levb — APPROVED** [review]
please fix go.mod

File: go.mod (hunk 2 above)
Comment (mickmister) [reply go.mod:15]: @levb I was waiting on https://github.com/mattermost/mattermost-plugin-apps/pull/304 to be merged so I can point to the merge commit. Do you suggest another path here?

Comment (mickmister) [reply go.mod]: Fixed

**Review by hanzei — APPROVED** [review]

**Pull request merged**